### ISSUE TITLE
Fail to import deconstructed object variable with Literals declaration

### ISSUE THREAD

**Issue by edoardocavazza** [rollup/rollup-plugin-commonjs#303]
The plugin fails bundling a module which exports a variable from deconstruction with literal keys.
```
[!] (commonjs plugin) TypeError: extractors[prop.key.type] is not a function
```

**Example**
*module.js*
```js
const obj = {
    index: 0,
    1: 1,
    2: 2,
    3: 3,
};

let {
    index: a,
    1: b,
    2: c,
    3: d
} = obj;

module.exports = b;
```
*index.js*
```js
import b from './module.js';
```

It seems a `rollup-pluginutils` failure (here the related [issue](https://github.com/rollup/rollup-pluginutils/issues/25) and [PR](https://github.com/rollup/rollup-pluginutils/pull/27)). Also, the [`extractNames`](https://github.com/rollup/rollup-plugin-commonjs/blob/master/src/ast-utils.js#L34) helper in `rollup-plugin-commonjs` could have the same issue.

**Comment by ispyinternet** [comment]
Looks like I'm hitting this when trying to rollup sequelize, the code in particular:

```
 static _defaultsOptions(...args) {
         return this._baseMerge(...args, (...cbArgs) => {
          [cbArgs[0], cbArgs[1]] = [cbArgs[1], cbArgs[0]];
          return this._mergeFunction(...cbArgs);
        });
  }
```

https://github.com/sequelize/sequelize/blob/4ec86a4bd43aa65e45546e2d4177ffcbeb2b4bac/lib/model.js#L817

**Comment by lukastaegert** [comment]
This particular issue should be fixed in the latest release via #358

**Comment by ispyinternet** [comment]
I don' think it is because the call to 

```
MemberExpression(names, node) {
		extractors[node.property.type](names, node.property);
	}
```

is made where `node.property.type = 'Literal'` and this method doesn't exist on the extractors object.
